README.md

```diff
@@ -1,6 +1,35 @@
 
 # Flexbox playground 
-[![All Contributors](https://img.shields.io/badge/all_contributors-9-orange.svg?style=flat-square)](#contributors)
+[![All Contributors](https://img.shields.io/badge/all_contributors-10-orange.svg?style=flat-square)](#contributors)
+
+
+# How to install yarn
+
+## Install yarn on Windows
+
+There are three options for installing Yarn on Windows, you can fine <a href="https://yarnpkg.com/lang/en/docs/install/#windows-stable" target="_blank">here</a> 
```

Comment:
Woops, typo here: This should be `you can find` instead of `you can fine` :)

Reply:
Ok Gracias, ya corregir el tema del fine, adicional agregue como instalar las dependencias con yarn y probar si se tiene instalado yarn con el yarn --version, gracias